v2/sourcedb-to-spanner/src/main/java/com/google/cloud/teleport/v2/source/reader/io/jdbc/rowmapper/provider/MysqlJdbcValueMappings.java

```diff
@@ -71,7 +71,7 @@ public class MysqlJdbcValueMappings implements JdbcValueMappingsProvider {
 
   /* Hex Encoded string for blob types. */
   private static final ResultSetValueMapper<Blob> blobToHexString =
-      (value, schema) -> Hex.encodeHex(value.getBytes(1, (int) value.length()));
```

Comment:
We might have to do this for binary and varbinary types also?

Reply:
Thanks for the catch! Done.